### DISCUSSION TITLE
.d.ts files that support the snappyjs library? Or I'm ready to submit a PR to your repository?

### DISCUSSION THREAD

**Discussion by LIUHAIK** [DefinitelyTyped/DefinitelyTyped#68897] [Request a new @types package]
.d.ts files that support the snappyjs library? :https://github.com/zhipeng-jia/snappyjs
Or I'm ready to submit a PR to your repository?

**Comment by MysteryBlokHed** [comment]
I just opened #68995

**Reply by MysteryBlokHed** [reply]
PR was just merged
